addons/udes_stock/models/stock_picking.py

```diff
@@ -2598,6 +2598,71 @@ def search_for_pickings(self, picking_type_id, picking_priorities, limit=1, doma
 
         return picking
 
+    def _find_unfulfillable_moves(self, batch_size=1000):
+        """Find unfullfilable order lines due to lack of stock."""
+        # This method is patterned on SaleOrder._find_unfulfillable_order_lines
+        # in udes_sale_stock.  Changes applied there should be considered for
+        # this method too.
+        Location = self.env["stock.location"]
+        MoveLine = self.env["stock.move"]
+        Quant = self.env["stock.quant"]
+
+        # Create empty record sets for SO lines
+        unfulfillable_lines = MoveLine.browse()
+
+        _logger.info("Checking pickings to cancel due to stock shortage")
```

Comment:
Message needs updating, we are not cancelling any picking. (original message from SaleOrder._find_unfulfillable_order_lines is probably incorrect too)